sections/staking/components/BurnTab/BurnTab.tsx

```diff
@@ -235,6 +242,13 @@ const BurnTab: React.FC = () => {
 				inputValue = toBigNumber(amountToBurn);
 				isLocked = false;
 				break;
+			case BurnActionType.CLEAR:
+				onBurnChange(maxBurnAmount.toString());
+				handleSubmit = () => handleClear();
+				inputValue = maxBurnAmount;
+				isLocked = true;
+				canClearDebt = false;
```

Comment:
@clementbalestrat is this meant to be true? not sure if BurnTiles accept this prop yet

Reply:
it's false for now as clear debt is still marked as "coming soon"